Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM navikt/node-express:12.2.0
+
+WORKDIR /usr/src/app
+
+COPY package.json server.js ./
+
+COPY dist/ ./dist
+COPY node_modules/ ./node_modules
+COPY views/ ./views
+
+EXPOSE 8080
```

Comment:
Må sørge for at `process.env.PORT` også er satt til `8080` (i nais.yaml?) eller endre default port i `server.js` https://github.com/navikt/registrer-tilretteleggingsbehov/blob/master/server/server.js#L5

Reply:
Vi pleier vel kjøre 3000 som default port